docs/releases/5.0.txt

```diff
@@ -265,7 +265,8 @@ Minor features
 :mod:`django.contrib.sessions`
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-* ...
+* An error message will be logged to the ``django.contrib.sessions`` logger
+  in case a cache storage failure for ``cahced_db``.
```

Comment:
Given that Django 5.0 was already released on December 4th, this release note should be moved to the 5.1.txt release notes, and it should be written in present tense (see the other notes as reference).
Also, there is a typo here:
```suggestion
  in case a cache storage failure for ``cached_db``.
```